hermes-console-vue/src/views/admin/constraints/create-constraint-form/CreateConstraintFormView.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { describe, expect } from 'vitest';
+import { render, renderWithEmits } from '@/utils/test-utils';
+import { waitFor } from '@testing-library/vue';
+import CreateConstraintFormView from '@/views/admin/constraints/create-constraint-form/CreateConstraintFormView.vue';
+
+describe('CreateConstraintForm', () => {
+  it('should emit a cancel event when user clicks cancel button', async () => {
+    // given
+    const wrapper = renderWithEmits(CreateConstraintFormView, {
+      props: {
+        subscription: false,
+      },
+    });
+
+    await wrapper
```

Comment:
please add given/when/then, also refers to another tests in this test class

Reply:
fixed